WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Multi-package/ShippingLabelPackageItem.swift

```diff
@@ -0,0 +1,48 @@
+import SwiftUI
+
+struct ShippingLabelPackageItem: View {
```

Comment:
Thanks a lot for the suggestion! I agree that `item` does sound ambiguous, but I'm not sure about `entity` and `unit` either 😓 I'll keep this in mind and update in a later PR, since renaming here can cause issues for the subsequent PRs.